_posts/2021-07-15-Subnero-wins-DisTec.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Subnero wins DisTec Undersea Challenge
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-15
+categories: news wnc
+thumbnail: images/pulse-thumbnail-distec.jpeg
+---
+
+We are delighted to announce that we won the [DisTec Undersea Challenge](https://www.udt-global.com/about-the-challenge){:target="_blank"}.
+
+The DisTec Undersea Challenge welcomed innovators and progressive organisations to showcase their solutions and disrupt the undersea defence environment as we know it.
+
+Our team submitted a 5 minutes [video](https://www.youtube.com/watch?v=zkaCsabBDoA&t){:target="_blank"} about the Subnero Underwater Network (SUN), which saw us selected for the Grand Finals, yesterday, along side of Marcom Defence and [Seacraft](https://seacraft.eu/){:target="_blank"}.
```

Comment:
Suggest we do not link to the video as that video will be taken down shortly. It is not meant for wider circulation, only to be used till we have a replacement video.

Reply:
Ok, I'll amend accordingly.